ddtrace/span.py

```diff
@@ -272,7 +274,7 @@ def set_tag(self, key, value=None):
             return
 
         # Key should explicitly be converted to a float if needed
-        elif key in NUMERIC_TAGS:
+        elif key in NUMERIC_TAGS and value:
```

Comment:
I would put the check for `value` first as it's probably just slightly quicker than `key in NUMERIC_TAGS`

Reply:
Right! Also, we should probably replace the optional value in this function since it's only used in one case when setting the measured span tag.